pkgs/development/php-packages/phpspy/default.nix

```diff
@@ -12,16 +13,23 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "phpspy";
-  version = "0.6.0";
+  version = "0.7.0";
 
   src = fetchFromGitHub {
     owner = "adsr";
     repo = "phpspy";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-iQOeZLHRc5yUgXc6xz52t/6oc07eZfH5ZgzSdJBcaak=";
+    hash = "sha256-QphoDdnSFPVRvEro0WDUC/yRsOf4I5p5BpHq32olqJI=";
     fetchSubmodules = true;
   };
 
+  patches = [
```

Comment:
Wouldn't this be better off using a `substituteInPlace` in `postPatch` ?

Reply:
I submitted the patch upstream, this is the reason why I use it here too. I would have used `substituteInPlace` if there hasn't been any patch for sure.